packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,22 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessible label for the checkbox. Defaults to 'label' property value.
```

Comment:
There is no code that assigns a default value to the ariaLabel property defined here, contrary to what the comment states.